src/Lesson1/Test.java

```diff
@@ -4,24 +4,20 @@
 
 
     public static void main(String[] args) {
+
         Cat cat = new Cat(100,2);
         Man man = new Man(1000,1);
         Robot robot = new Robot(10000,10);
         Wall wall = new Wall(10);
         Treadmill treadmill = new Treadmill(200);
 
-        Do [] doing = {cat, man, robot};
-
+        Game game = new Game();
+        game.setObstacles(man, robot, cat);
+        game.setParticipants(wall, treadmill);
```

Comment:
поменять местами параметры в методах
```suggestion
        game.setParticipants(man, robot, cat);
        game.setObstacles(wall, treadmill);
```